dll/win32/msgina/gui.c

```diff
@@ -56,28 +56,34 @@ DlgData_Create(HWND hwndDlg, PGINA_CONTEXT pgContext)
     return pDlgData;
 }
 
-static BOOL
-DlgData_LoadBitmaps(PDLG_DATA pDlgData)
+static VOID
+DlgData_LoadBitmaps(_Out_ PDLG_DATA pDlgData)
```

Comment:
Technically this is `_Inout_`

Reply:
Right, I missed to account for the `pDlgData->pgContext->hDllInstance` read. Fixed.